src/org/gk/util/BrowserLauncher.java

```diff
@@ -96,15 +96,7 @@ public static String getBrowser() {
 	}
 	
 	private static File getPropertyFile() {
-	    File file = null;
```

Comment:
Joel, Actually I'd like to make a change in class GKApplicationUtilities:

	 public static File getPropertyFile(String fileName) throws IOException {
	     File file = new File(REACTOME_DIR + File.separator + fileName);
	     return file;
	 }

and add a check to make this this property exits, which is very important before return:

if (!file.exist()) throw new FileNotFoundException. 

By doing this, we don't need to make other changes either.

Reply:
Okay, I've added in the logic for the FileNotFoundException and reverted the changes to the try/catch block that was removed here: [ef8dfd](https://github.com/reactome/CuratorTool/pull/14/commits/ef8dfd6652fa8dea67b97ded1bfd16c10146e930?w=1)